### PR TITLE
 CRM-19718 Activities: make priority_id available for profiles

### PR DESCRIPTION
**Pull request by alifrumin**
* [CRM-19718: Allow activity Priority field to be included in profiles](https://issues.civicrm.org/jira/browse/CRM-19718)

### CODE REVIEW

**Comment by alifrumin** [comment]
This is the 4.6 version of #9508

**Comment by jackrabbithanna** [comment]
Jenkins test this please

**Comment by jackrabbithanna** [comment]
I know its been a while, but I'm looking through the PR queue now...
@alifrumin by looking at the issue, could use your thoughts if I am to merge this

https://issues.civicrm.org/jira/browse/CRM-19718

It looks like there was an additional PR needed for this issue....

I'm hesitant to merge this without knowing if it needed more
https://github.com/civicrm/civicrm-core/pull/9665

**Comment by alifrumin** [comment]
Hey @jackrabbithanna, sorry for the delay, I will take a look at this today

**Comment by alifrumin** [comment]
My memory around this is a little hazy at this point.

I think what happened here is that I made a pr to 4.6 (this pr) and 4.7 (#9508) to deal with bug CRM-19718. Then @yashodha opened an alternate pr (that accomplishes the same thing but in a more comprehensive way) https://github.com/civicrm/civicrm-core/pull/9665 for 4.7 that has not yet been merged for 4.7. 

We are using this fix for one of our clients and have not had problems, but it may be better to use a more comprehensive fix like @yashodha wrote for master.

File: xml/schema/Activity/Activity.xml [hunk 1]
```diff
@@ -266,6 +266,7 @@
     <name>priority_id</name>
     <type>int unsigned</type>
     <title>Priority</title>
+    <import>true</import>
```
Comment (monishdeb) [inline xml/schema/Activity/Activity.xml]: @alifrumin can you also add this property in [Activity DAO](https://github.com/civicrm/civicrm-core/blob/master/CRM/Activity/DAO/Activity.php#L439)  ?

**Comment by jackrabbithanna** [comment]
I've been trying to test this, I'm confused

When I add the Activity Priority field to a profile used by an event, I get an error with or without this code...

When I add Activity Priority to a profile used on a contribution page, i get a text field for the activity, no errors...
After this PR, it continues to show as a text field, no select list...

What am I missing here?

What are the steps to reproduce your problem?

**Comment by alifrumin** [comment]
Im not sure maybe this fix is dependent on other changes in 4.7 I will close it. sorry for the confusion.

**Pull request closed without merging**